src/components/postDetail/ImageSlide/ImageSlide.module.scss

```
@@ -0,0 +1,84 @@
+%btn-base {
+  @include pos-center-y;
+
+  width: 4rem;
+  height: 4rem;
+  padding: 1rem;
+
+  background: $opacity-white-10;
+  backdrop-filter: $popup-blur;
+  border: 0.1rem solid $opacity-white-20;
+  border-radius: 0.4rem;
+}
+
+$container-width: 100vw;
+$mobile-image-width: 70vw;
+
```

SCSS 파일중에 `_responsive-height.scss` 파일이 있습니다.
반응형일 때 변경되는 높이 값을 선언해서 사용하는 파일인데요, 너비도 추가될 수 있을 것 같습니다.
해당 파일 이름을 responsive-values.scss 변경하고 우림님께서 선언하신 변수도 함께 넣으면 좋을 것 같습니다~
```suggestion

$slider-container-width: 100vw;
$mobile-image-width: 70vw;

```

---

슬라이드 배너가 부드럽게 옆으로 넘어가는 것이 아니라 버튼 누를 때 딱딱 화면만 변경되는 느낌입니다.
슬라이드를 부드럽게 이동되게 변경해 주시면 좋을 것 같습니다~

---

옙!